linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_write_logs/rule.yml

```diff
@@ -18,6 +18,7 @@ identifiers:
 
 references:
     ospp: FAU_GEN.1.1.c
+    srg: SRG-OS-000051-GPOS-00024
```

Comment:
Nack. ``SRG-OS-000051-GPOS-00024`` says "The operating system must provide the capability to centrally review and analyze audit records from multiple components within the system."

RHEL provides this capability and cannot be configured to be out of compliance with providing the capability. Should be marked as "inherently met."

Reply:
The rule sets the configuration so that auditd will write logs to disk.  Without doing that, central review and analysis would be difficult.

Let me know if you agree or not.  I can change if you do not agree.

Reply:
I am not native English speaker, SRG specialist nor lawyer. But I would interpret "The OS must provide the capability" as "you are able, with the tools provided by the operating system, to achieve <that>". I am with Shawn here.

Reply:
The write logs option creates a local log in case someone wants to investigate something on the machine rather than in the central facility.

Reply:
Good rule to have. Remapped to SRG-OS-000480-GPOS-00227